space2stats_api/app/settings.py

```diff
@@ -8,8 +8,8 @@ class Settings(BaseSettings):
     DB_USER: str
     DB_PASSWORD: str
     DB_TABLE_NAME: str
-
-    model_config = SettingsConfigDict(env_file="../local_db.env")
+        
+    model_config = SettingsConfigDict(env_file='../db.env')
```

Comment:
why do we have to put `db.env` in the top level?  we could put this in https://github.com/worldbank/DECAT_Space2Stats/blob/main/space2stats_api

Reply:
moved!